### DISCUSSION TITLE
how make two signatures area in the same page

### DISCUSSION THREAD

**Discussion by mediaboost** [creagia/laravel-sign-pad#9] [Q&A]
Hello

My question is in the title : how make two pad sign in the same page ?

there is an js conflict and the second pad not run

thank's a lot

**Comment by dtorras** [comment]
Thanks for this report. Please, open an issue with the details, and we'll look into it.

**Comment by dtorras** [comment]
We've just released a new major version that fixes this issue: https://github.com/creagia/laravel-sign-pad/releases/tag/2.0.0
